example/example.go

```diff
@@ -60,25 +60,10 @@ func main() {
 `
 	output := template.Must(template.New("outputPage").Parse(tmpl))
 
-	fmt.Println("Generating front-end html.")
-
-	c := &gobra.CommandFromCobra{cmd.Root, ""}
-
-	val, err := c.Render()
-	if err != nil {
-		panic(err)
-	}
-
-	f, err := os.Create("index.html")
-	if err != nil {
-		panic(err)
-	}
-	output.Execute(f, template.HTML(val))
+	fmt.Println("Starting server at localhost:8080")
 
-	fmt.Println("Successfully generated front-end html.")
-	fmt.Println("Starting server, with front-end html/js.")
-	fmt.Println("------------------")
+	c := &gobra.CommandFromCobra{CobraCmd: cmd.Root, ServerAddress: "localhost:8080"}
 
-	server := gobra.Server{cmd.Root, 8080, false, false}
+	server := gobra.Server{Root: c, AllowCORS: true, HTML: output}
```

Comment:
The PR looks good so I merged it. Do you think we should reduce redundancy even further by not having gobra.CommandFromCobra at all? Since it's going to be used in gobra.Server anyways and you've made .render() a private method.